packages/survey-creator-core/src/components/creator.scss

```diff
@@ -8,6 +8,7 @@ survey-creator,
   position: relative;
   z-index: 0;
 
+  -webkit-font-smoothing: antialiased;
```

Comment:
-webkit prefix AFAIR works for chrome only. Do we need to fix the issue in Safari also?

Reply:
No, I checked it in Safari. We already use the same solution on our website and it works as expected.